tests/unit_test/data_processor/data_processor_test.py

```diff
@@ -281,12 +284,12 @@ def test_add_prompt_action_with_invalid_query_prompt(self):
                                     'instructions': 'Answer according to the context', 'type': 'query',
                                     'source': 'history', 'is_enabled': True},
                                    {'name': 'Query Prompt',
-                                    'data': 'If there is no specific query, assume that user is aking about java programming.',
-                                    'instructions': 'Answer according to the context', 'type': 'query',
-                                    'source': 'history', 'is_enabled': True}],
+                                    'data': 'email',
+                                    'instructions': '', 'type': 'query',
+                                    'source': 'slot', 'is_enabled': True}],
                    "failure_message": DEFAULT_NLU_FALLBACK_RESPONSE, "top_results": 10, "similarity_threshold": 0.70,
                    "num_bot_responses": 5}
-        with pytest.raises(ValidationError, match="Query prompt must have static source!"):
```

Comment:
lets bring this check back: Query prompt must have static source!
remove this one: instructions are required for type query!